byzcoin/viewchange/viewchange.go

```diff
@@ -125,7 +127,9 @@ func (c *Controller) Stop() {
 }
 
 // Start begins a blocking process that processes incoming view-requests.
-func (c *Controller) Start(myID network.ServerIdentityID, genesis skipchain.SkipBlockID, initialDuration time.Duration, f int) {
```

Comment:
Same changes already merged in https://github.com/dedis/cothority/pull/2268 (and some more below)

Reply:
Yes, true. I re-based it on #2268... I'll rebase it on master.